src/airdrop.ts

```diff
@@ -326,6 +335,83 @@ export class Airdrop {
     };
   }
 
+  /**
+   * Create a transaction with the instructions for setting up a governance
+   * airdrop. This includes the config as well as transferring the tokens.
+   */
+  public async createConfigGovernanceTransaction(
+    source: PublicKey,
+    authority: PublicKey,
+    totalAmount: BN,
+    amountPerVoter: BN,
+    eligibilityStart: BN,
+    eligibilityEnd: BN,
+    governance: PublicKey,
+  ): Promise<AirdropConfigureContext> {
+    const transaction: Transaction = new Transaction();
+
+    const { mint } = await getAccount(this.connection, source, 'single');
+
+    const airdropSeed = crypto.randomBytes(32);
+    const verifierSeed = crypto.randomBytes(32);
+    const [governanceAirdropState, _governanceAirdropStateBump] = (
+      web3.PublicKey.findProgramAddressSync(
+        [airdropSeed],
+        this.airdropProgram.programId,
+      ));
+    const [governanceVerifierState, _governanceVerifierBump] = (
+      web3.PublicKey.findProgramAddressSync(
+        [verifierSeed],
+        this.governanceVerifierProgram.programId,
+      ));
+    const governanceVault = this.getVaultAddress(governanceAirdropState);
+    const [verifierSignature, _signatureBump] = web3.PublicKey.findProgramAddressSync(
+      [governanceAirdropState.toBuffer()],
+      this.governanceVerifierProgram.programId,
+    );
+
+    const governanceConfigureIx = await this.airdropProgram.methods.configure(
+      airdropSeed,
+    )
+      .accounts({
+        payer: authority,
+        verifierSignature,
+        vault: governanceVault,
+        mint,
+        state: governanceAirdropState,
+        tokenProgram: TOKEN_PROGRAM_ID,
+        systemProgram: web3.SystemProgram.programId,
+        rent: web3.SYSVAR_RENT_PUBKEY,
+      })
+      .instruction();
+
+    // First instruction configures the airdrop program.
+    transaction.add(governanceConfigureIx);
+
+    const governanceInitIx = await this.governanceVerifierProgram.methods
+      .configure(verifierSeed, amountPerVoter, eligibilityStart, eligibilityEnd)
+      .accounts({
+        payer: authority,
+        state: governanceVerifierState,
+        airdropState: governanceAirdropState,
+        governance,
+        systemProgram: web3.SystemProgram.programId,
+      })
+      .instruction();
+
+    // Next instruction configures the governance verifier.
+    transaction.add(governanceInitIx);
```

Comment:
Does this force the program to only be able to verify claims after the fact? We cannot set it to say if any of the next 3 proposals are voted on then your eligible?